ide/db/src/org/netbeans/modules/db/explorer/node/Bundle.properties

```diff
@@ -33,7 +33,7 @@ ND_ForeignKey=Foreign Key
 ND_PrimaryKey=Primary Key
 ND_Index=Index
 
-ND_ProcedureList=List of Stored Procedures
+ND_ProcedureList=List of Stored Routines
```

Comment:
Please keep the original name if there not evidence, that other DBMS apart from mysql use Routine instead of procedure. If there is evidence, please elaborate.